timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -97,9 +99,10 @@
     private static final TimeLockServerHolder TIMELOCK_SERVER_HOLDER =
             new TimeLockServerHolder(TEMPORARY_CONFIG_HOLDER::getTemporaryConfigFileLocation);
 
-    private static final LockRequest SLOW_REQUEST = LockRequest
+    private static final LockRequest REQUEST_LOCK_WITH_LONG_TIMEOUT = LockRequest
             .builder(ImmutableSortedMap.of(StringLockDescriptor.of("lock"), LockMode.WRITE))
-            .blockForAtMost(SimpleTimeDuration.of(200, TimeUnit.MILLISECONDS))
+            .timeoutAfter(SimpleTimeDuration.of(10, TimeUnit.MINUTES))
```

Comment:
We might want the timeout to be shorter, in case this test gets stuck for some reason.